site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -45,6 +46,29 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const editThread = (title) => {
+    cy.get('.thread-left-cont > .thread-list-item').contains(title).click();
+    cy.get('.create-post-head').should('contain', title);
+    cy.get('.first_post > .post-action-container > .edit-post-button').click();
+    cy.get('#upload1').find('[for="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
+    cy.get('#file-upload-table-1').find('.file-label').should('contain', 'sea_animals.png');
+    cy.get('#file-upload-table-1').find('[aria-label="Press enter to remove file sea_animals.png"]').click();
```

Comment:
Finding an element by aria-label can be fragile, could you make test-ids for the other elements being tested that are new in this PR?